lib/new_relic/agent/llm/chat_completion_summary.rb

```diff
@@ -24,6 +32,13 @@ def attributes
           LlmEvent::ATTRIBUTES + ChatCompletion::ATTRIBUTES + ResponseHeaders::ATTRIBUTES + ATTRIBUTES
         end
 
+        def attribute_name_exceptions
+          # TODO: OLD RUBIES < 2.6
+          # Hash#merge accepts multiple arguments in 2.6, so we can reduce this
+          # to a single Hash#merge call with two arguments at that point
```

Comment:
This is where better granular perf tests would help. If there's a significant performance gain to be had, it'd be worth writing the code both ways:

```ruby
if RUBY_VERSION >= 2.6
  new_way
else
  old way
end
```

Reply:
Great call out! Ran a perf test a few times and `item.merge(item1).merge(item2)` is consistently ~1.25x slower. We will break this out into a conditional.

Reply:
Done! I wanted to use a ternary operator but it got super long https://github.com/newrelic/newrelic-ruby-agent/pull/2398/commits/36d6cab921169da77ee3bccd2e83fd0da3ea728c